### PR TITLE
Added new exercise: Knapsack

### LINKED ISSUE #917: Add new exercise: knapsack
[problem specification ](https://github.com/exercism/problem-specifications/tree/master/exercises/knapsack)| [description](https://github.com/exercism/problem-specifications/blob/master/exercises/knapsack/description.md) | [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/knapsack/canonical-data.json)

You may find it helpful to follow [this checklist](https://github.com/exercism/javascript/blob/83feda2af955df1ef9f3a8080338f506f0fdf696/CONTRIBUTING.md#new-exercise) from the [CONTRIBUTING.md](https://github.com/exercism/javascript/blob/83feda2af955df1ef9f3a8080338f506f0fdf696/CONTRIBUTING.md) resource.

### PR DESCRIPTION
**Pull request by lpizzinidev**
closes #917

### CODE REVIEW

File: exercises/practice/knapsack/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@exercism/javascript-knapsack",
+  "description": "Exercism exercises in Javascript.",
+  "author": "lpizzinidev",
```
Comment (SleeplessByte) [inline exercises/practice/knapsack/package.json:4]: Not a necessity, but you may want to register (for free) on npmjs: https://www.npmjs.com/~lpizzinidev, so people can find you :)

**Review by SleeplessByte — APPROVED** [review]
cc @junedev You agree with the prereqs?

**Comment by junedev** [comment]
I think the prerequisites are fine for now. We can review them at some later point once we have some community solutions to look at (or change them in case someone complains :wink:).

**Review by junedev — APPROVED** [review]

File: exercises/practice/knapsack/package.json (hunk 1 above)
Comment (lpizzinidev) [reply exercises/practice/knapsack/package.json:4]: Thanks for the suggestion, I will register 👍🏻

**Comment by SleeplessByte** [comment]
*Help Bob the thief :moneybag:*
Bob is a thief. After months of careful planning, he finally manages to crack the security systems of a high-class apartment. Help Bob solve the classic knapsack problem by helping Bob determine the maximum value he can get from the items in the house.

**Pull request merged**